### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by marcelklehr** [nextcloud/recognize#989] [Announcements]
### Breaking changes
 - Drop support for Nextcloud 26

### New
- storage crawl job: Increase batch size to 2000
- Allow enabling/disabling classifier concurrency

### Fixed
 - fix(IgnoreService): Always return paths matching filecache format (fixes ignore files in external storage)
 - Remove max execution time limitation and disable parallel classifier job execution (fixes stuck classification)
 - Fix(ViewAdmin): Don't display last bg job execution if it's 0
 - Fix(snap): Improve interoperability with nc snap
 - fix(AdminController): Catch failed cpuinfo readfile
 - Fix(l10n): Update translations from Transifex

<hr /><em>This discussion was created from the release <a href='https://github.com/nextcloud/recognize/releases/tag/v5.0.0'>v5.0.0</a>.</em>

**Comment by marcelklehr** [comment]
Ahoy everyone! This release comes with fixes for quite a lot long standing issues in recognize and fur that reason had to drop support for Nextcloud 26. This is thus recognize v5 :tada:

**Comment by DenuxPlays** [comment]
I tried to update recognize but I am getting the following exception.
But I am not sure if this is caused by nextcloud AIO or by recognize.
BTW. I am running nextcloud AIO version 7.4.0
```
[no app in context] Error: Exception: Could not extract app recognize: Out-of-path file extraction {/tmp/oc_tmp_pKedcn-folder/recognize/node_modules/@tensorflow/tfjs-node/build-tmp-napi-v8/node_gyp_bins/python3 --> /bin/python3} at <<closure>>

0. /var/www/html/lib/private/Installer.php line 193
   OC\Installer->downloadApp("*** sensitive parameters replaced ***", false)
1. /var/www/html/apps/settings/lib/Controller/AppSettingsController.php line 535
   OC\Installer->updateAppstoreApp("*** sensitive parameters replaced ***")
2. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 230
   OCA\Settings\Controller\AppSettingsController->updateApp("*** sensitive parameters replaced ***")
3. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 137
   OC\AppFramework\Http\Dispatcher->executeController(["OCA\\Settings\ ... "], "updateApp")
4. /var/www/html/lib/private/AppFramework/App.php line 183
   OC\AppFramework\Http\Dispatcher->dispatch(["OCA\\Settings\ ... "], "updateApp")
5. /var/www/html/lib/private/Route/Router.php line 315
   OC\AppFramework\App::main("OCA\\Settings\\ ... r", "updateApp", ["OC\\AppFramewo ... "], ["*** sensitive  ... "])
6. /var/www/html/lib/base.php line 1068
   OC\Route\Router->match("/settings/apps/update/recognize")
7. /var/www/html/index.php line 36
   OC::handleRequest()

GET /settings/apps/update/recognize
from 193.158.181.116 by admin at 2023-10-08T13:28:41+00:00
```

**Reply by marcelklehr** [reply]
Should be fixed with v5.0.1
